addons/binding/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/handler/TeslaHandler.java

```diff
@@ -716,9 +716,11 @@ private ThingStatusDetail authenticate() {
 
         Response response = tokenTarget.request().post(Entity.entity(payLoad, MediaType.APPLICATION_JSON_TYPE));
 
-        logger.debug("Authenticating : Response : {}:{}", response.getStatus(), response.getStatusInfo());
+        if (response == null) {
```

Comment:
The old case would throw an NPE in the debug log statement before reaching the if.